modules/cli/pages/index.adoc

```diff
@@ -6,6 +6,11 @@ A Command Line Interface tool (CLI) is used to provision data extracted from Bon
 
 The command line tool is delivered as a package, containing a basic documentation about commands and a sample configuration file.
 
+=== How to obtain the tool?
+
+[example]
+To obtain the export tool, ask your Bonitasoft contact
```

Comment:
contact a sales representative using this form:  [contact us](https://www.bonitasoft.com/contact-us)

Reply:
tool should be available on CSC for existing customers

Reply:
For CSC access we need to wait until csc is ready